fronts-client/src/util/form.ts

```diff
@@ -63,8 +67,10 @@ export interface CapiFields {
   thumbnail?: string | void;
 }
 
-const strToInt = (str: string | void) => (str ? parseInt(str, 10) : undefined);
-const intToStr = (int: number | void) => (int ? int.toString() : undefined);
+export const strToInt = (str: string | void) =>
```

Comment:
No problems with this, but see my earlier comment that it may return `NaN` depending on the contents of the string and you might want to make sure that this is handled properly